cms/static/build.js

```diff
@@ -153,6 +153,6 @@
          * SILENT: 4
          * Default is 0.
          */
-        logLevel: 4
+        logLevel: 1
```

Comment:
Yes, I think so. It doesn't output too much at this level, and it means that you can get a sense for what it is doing. At the very least it should be set to 2 so that you can see warnings and errors, but I found that 1 was more useful.